test_project/pages/home_page.py

```diff
@@ -1,34 +1,22 @@
-from selenium.webdriver.common.by import By
-from selenium.webdriver.remote.webelement import WebElement
-from selenium.webdriver.support.wait import WebDriverWait
-from selenium.webdriver.support import expected_conditions as ec
+from playwright.sync_api import Page, expect
 
 from test_project.pages.base_page import BasePage
-from framework.utils.settins_utils import settings
-
-cookies_button_selector = (By.CSS_SELECTOR, '#onetrust-accept-btn-handler')
-search_button_selector = (By.CSS_SELECTOR, 'a[href="/search"]')
 
 
 class HomePage(BasePage):
-    def __init__(self, browser):
-        super().__init__(browser)
-
-    def __get_cookies_button(self) -> WebElement:
-        return self.find_element(cookies_button_selector)
-
-    def click_cookies_button(self):
-        self.__get_cookies_button().click()
+    def __init__(self, page: Page):
+        super().__init__(page)
+        self.cookies_button = page.get_by_role("button", name="Accept Cookies")
```

Comment:
items would be searched during the page init, isn't it? would the init fail if any of them missing?

Reply:
fixed